derive_builder/src/lib.rs

```diff
@@ -178,6 +178,34 @@
 //! - `#[builder(setter(skip = "true"))]`
 //! - `#[builder(setter(skip = "false"))]`
 //!
+//! ## Custom setters (skip autogenerated setters)
+//!
+//! Similarly to `setter(skip)`, you can say that you will provide your own setter methods.
+//! This simply suppresses the generation of the setter, leaveing the field in the builder,
+//! as `Option<T>`.
+//!
+//! ```rust
+//! # #[macro_use]
+//! # extern crate derive_builder;
+//! #
+//! #[derive(Builder)]
+//! struct SetterOptOut {
+//!     #[builder(setter(custom))]
+//!     custom_setter: u32,
+//! }
+//! impl SetterOptOutBuilder {
+//!     fn set_custom_setter(&mut self, value: u32) {
+//!         self.custom_setter = Some(value);
+//!     }
+//! }
+//! # fn main() {}
+//! ```
+//!
+//! Again, the more verbose form is accepted:
+//!
+//! - `#[builder(setter(custom = "true"))]`
```

Comment:
I don't think quotes are needed here.

Reply:
Ah yes.  I got that from the `skip` example.  I've removed the quotes from both and amended one of the tests to exercise this case.